File: fabric/src/main/java/juuxel/adorn/compat/TechRebornCompat.java
package juuxel.adorn.compat;

import juuxel.adorn.block.variant.BlockVariant;
import juuxel.adorn.block.variant.CompatBlockVariantSet;

import java.util.List;

public final class TechRebornCompat implements CompatBlockVariantSet {
    @Override
    public String getModId() {
        return "techreborn";
    }

    @Override
    public List<BlockVariant> getWoodVariants() {
        return List.of(new BlockVariant.Wood("techreborn/rubber"));
    }
}

File: fabric/src/main/kotlin/juuxel/adorn/compat/TechRebornCompat.kt
package juuxel.adorn.compat

import juuxel.adorn.api.block.BlockVariant
import juuxel.adorn.block.BlockVariantSet

object TechRebornCompat : BlockVariantSet {
    override val woodVariants = listOf(BlockVariant.Wood("techreborn/rubber"))
}
